README.md

```diff
@@ -26,6 +26,33 @@ To run tests, run `rake db:test:prepare; rspec`.
 
 You'll need to configure cloudinary, [including adding a cloudinary.yml file](http://cloudinary.com/documentation/rails_integration).
 
+#### Running Frolfr in Docker
+If process isolation is your bad then you're in luck - the Frolfr API is full containerized! Make sure you have docker fully setup and running in your local environment and then do the following to get frolfr running:
```

Comment:
supposed to be `bag`
![source](https://cloud.githubusercontent.com/assets/7738688/22627368/fc21e2b4-eb8f-11e6-988e-90b281896e9c.gif)